alternator/executor.cc

```diff
@@ -930,6 +930,12 @@ static future<executor::request_return_type> create_table_on_shard0(tracing::tra
         co_return api_error::validation(format("Prefix {} is reserved for accessing internal tables", executor::INTERNAL_TABLE_PREFIX));
     }
     std::string keyspace_name = executor::KEYSPACE_NAME_PREFIX + table_name;
+    if (!request.HasMember("AttributeDefinitions")) {
+        co_return api_error::validation("No Attribute Schemas Defined");
+    }
+    else if (!request.HasMember("attributeDefinitions")) {
+        co_return api_error::validation("No Attribute Schemas Defined");
+    }
```

Comment:
This code is wrong - any CreateTable request that doesn't  have BOTH AttributeDefinitions and attributeDefinitions (!) will throw this error. This is obviously a mistake and you should have seen it causing any Alternator test to fail, so please do try to run Alternator tests before sending the next version of the PR (just run `test/alternator/run`, and in about 30 seconds you will have run the entire Alternator test suite).

I also don't understand what you were **trying** to do here. Do you suspect that DynamoDB allows also the capitalization attributeDefinitions and want to support it in Alternator as well? Are you sure that DynamoDB allows it? Did you check it?

Reply:
I'll test again